config.go

```diff
@@ -36,10 +36,11 @@ type NetworkConfig struct {
 }
 
 type FileConfig struct {
-	Paths    []string          `json:paths`
-	Fields   map[string]string `json:fields`
-	DeadTime string            `json:"dead time"`
-	deadtime time.Duration
+	Paths        []string          `json:paths`
+	Fields       map[string]string `json:fields`
+	DeadTime     string            `json:"dead time"`
+	deadtime     time.Duration
+	ExcludePaths []string `json:"exclude"`
```

Comment:
ExcludePaths differs from the config file param "exclude". Maybe should be "Exclude" in the struct for consistency.